packages/stargate/src/stargateclient.ts

```diff
@@ -63,6 +59,7 @@ export interface IndexedTx {
   readonly code: number;
   readonly rawLog: string;
   readonly tx: Uint8Array;
+  readonly gasUsed: number;
```

Comment:
Can we add gasWanted here as well? This should be trivial to get now and probably makes some user happy